autotest/t081_test.py

```diff
@@ -0,0 +1,350 @@
+"""
+created matt_dumont 
+on: 15/02/22
+"""
+import flopy
+import numpy as np
+from ci_framework import FlopyTestSetup, base_test_dir
+import platform
+
+base_dir = base_test_dir(__file__, rel_path="temp", verbose=True)
+nrow = 3
+ncol = 4
+nlay = 2
+nper = 1
+l1_ibound = np.array([[[-1, -1, -1, -1],
+                       [-1, 1, 1, -1],
+                       [-1, -1, -1, -1]]])
+l2_ibound = np.ones((1, nrow, ncol))
+l2_ibound_alt = np.ones((1, nrow, ncol))
+l2_ibound_alt[0, 0, 0] = 0
+ibound = {
+    'mf1': np.concatenate((l1_ibound, l2_ibound), axis=0),  # constant heads around model on top row
+    'mf2': np.concatenate((l1_ibound, l2_ibound_alt), axis=0),  # constant heads around model on top row
+}
+laytype = {
+    'mf1': [0, 1],
+    'mf2': [0, 0]
+}
+hnoflow = -888
+hdry = -777
+top = np.zeros((1, nrow, ncol)) + 10
+bt1 = np.ones((1, nrow, ncol)) + 5
+bt2 = np.ones((1, nrow, ncol)) + 3
+botm = np.concatenate((bt1, bt2), axis=0)
+ipakcb = 740
+
+names = ['mf1', 'mf2']
+
+mf2005_exe = "mf2005"
+if platform.system() in "Windows":
+    mf2005_exe += ".exe"
+mf2005_exe = flopy.which(mf2005_exe)
+
+mp6_exe = "mp6"
+if platform.system() in "Windows":
+    mp6_exe += ".exe"
+mp6_exe = flopy.which(mp6_exe)
+
+
+
+def make_test_modflow_model(nm, ws):
```

Comment:
No it is not a test, but rather some code to make a modflow model that is needed for the future tests. It inherits the ws as an arg.  I have renamed to _setup_modflow_model  (on next commit) to avoid confusion